components/dada-validate/src/validate/validator.rs

```diff
@@ -398,7 +398,8 @@ impl<'me> Validator<'me> {
             }
 
             syntax::ExprData::Parenthesized(parenthesized_expr) => {
-                self.validate_expr_in_mode(*parenthesized_expr, mode)
+                let validated_expr = self.validate_expr_in_mode(*parenthesized_expr, mode);
+                self.add(validated::ExprData::Parenthesized(validated_expr), expr)
```

Comment:
I had to change that because otherwise this assert was failing: https://github.com/dada-lang/dada/blob/60986a30ec5ccf130318335231ff4aabe28904c9/components/dada-validate/src/validate/validator.rs#L194-L196

Before adding `Parenthesized` to `Validated` I did https://github.com/dada-lang/dada/blob/e4c5bdc4866a17666206b9544a8b84edc2fa278a/components/dada-validate/src/validate/validator.rs#L400-L406
but that felt wrong. ¿Maybe that is a better fix?

Reply:
Ah, you mean the assertion in validator, not the one from the #141, right?
There's another one in `reserve_validated_expr`